lib/tasks/listen.js

```diff
@@ -133,7 +133,8 @@ class TaskListen extends Task {
       mixType: this.mixType,
       sampling: this.sampleRate,
       ...(this._bugname && {bugname: this._bugname}),
-      metadata
+      metadata,
+      bidirectionalAudio: this.bidirectionalAudio || {}
```

Comment:
disableBidirectionalAudio is there for legacy purposes, so we have to keep it.  But I think in this listen.js code we should immediately set `bidirectionalAudio.enabled` here and then in the rest of the code refer only to that, ie.right after the line above do something like
```
if ('enabled' not in this.bidirectionalAudio) {
   this.bidirectionalAudio.enabled = this. disableBidirectionalAudio === true ? false : true;
}
```

Reply:
are you going to address this?

Reply:
Yes doing it now

Reply:
fixed